Improvement of the Data column explanation + updating tenants PR (#88) (#164)
stancl/tenancy-docs: source/docs/v3/tenants.blade.md

## Custom columns {#custom-columns}

> Attributes of the tenant model which don't have their column will be stored in the `data` JSON column. You can set these attributes as you'd normally set model attributes:

Reviewer: - The first sentence doesn't need to be changed, add the `own` back
- updated second sentence
```suggestion
Attributes of the tenant model which don't have their own column will be stored in the `data` JSON column. You can set these attributes like you'd set normal model attributes:
```

```php
$tenant->update([
    'attributeThatHasNoColumn' => 'value', // stored in the `data` JSON column

>     'plan' => 'free' // stored in the `plan` column (see below)

Reviewer: ```suggestion
    'plan' => 'free' // stored in the dedicated `plan` column (see below)
```

]);
```
or
```php
$tenant->customAttribute = 'value'; // stored in the `data` JSON column
$tenant->plan = 'free'; // stored in the `plan` column (see below)
$tenant->save();
```

> You may define the custom columns that **won't** be stored in the `data` JSON column by overriding the `getCustomColumns()` method on your `Tenant` model:

Reviewer: ```suggestion
You may define the custom columns (that **won't** be stored in the `data` JSON column) by overriding the `getCustomColumns()` method on your `Tenant` model:
```
